lib/resque/job.rb

```diff
@@ -193,6 +194,8 @@ def perform
           job.send(hook, *job_args)
         end
 
+        @final_args = job_args
```

Comment:
I'm not sure how I feel about this. Is the idea that plugins or users that use this hook will then know to access `final_args` in their perform method?